stable/prometheus-blackbox-exporter/values.yaml

```diff
@@ -77,7 +80,7 @@ service:
 
 serviceAccount:
   # Specifies whether a ServiceAccount should be created
-  create: false
+  create: true
```

Comment:
Hm... I wonder if this is a breaking change. Technically speaking, yes, but I don't know if it would break any existing workflow...

Reply:
Well, could you bump the major version of the chart to advertise this breaking change? Actually, adding PSP is be a breaking change as well (it can break existing installations since it is enabled by default).